docs/runtime_suite/mailchimp-notification-service/20_configuration.md

```diff
@@ -0,0 +1,22 @@
+---
+id: configuration
+title: Configuration
+sidebar_label: Configuration
+---
+
+<!--
+WARNING: this file was automatically generated by Mia-Platform Doc Aggregator.
+DO NOT MODIFY IT BY HAND.
+Instead, modify the source file and run the aggregator to regenerate this file.
+-->
+
+The microservice uses the following environment variables:
+
+- `MANDRILL_API_KEY` (required): Mandrill API key to send transactional emails. Can be created on the Mandrill portal.
```

Comment:
```suggestion
The microservice uses the following environment variables:

- `MANDRILL_API_KEY` (required): Mandrill API key to send transactional emails. It Can be created on the Mandrill portal.
```